src/cli/remove.rs

```diff
@@ -44,40 +55,58 @@ pub async fn execute(args: Args) -> miette::Result<()> {
     let results = deps
         .iter()
         .map(|dep| {
-            if let Some(p) = &args.platform {
-                project
-                    .manifest
-                    .remove_target_dependency(dep, &spec_type, p)
+            if args.pypi {
+                let pkg_name = rip::types::PackageName::from_str(dep.as_source())
+                    .expect("Expected dependency name.");
```

Comment:
Just like with the `add` functionality I think this cli subcommand should accept strings for `deps`. Parsing into a package name should yield an error if its not a valid package name. That way the user gets the best possible error message.

Reply:
I refactored that, please take a look again :)